src/python/nightlyBench.py

```diff
@@ -1601,7 +951,7 @@ def makeGraphs():
             gcSearchTimes = getSearchGCTimes('%s/%s' % (constants.NIGHTLY_LOG_DIR, subDir))
             s = timeStampString
             for h in 'JIT compilation', 'Young Generation GC', 'Old Generation GC':
-                v = gcIndexTimes.get(h)
+                v = gcSearchTimes.get(h)
```

Comment:
Egads, this is not a small bug!!  This means all this time we've been looking at only index GC times, even in the [supposed Search GC/JIT time chart](https://home.apache.org/~mikemccand/lucenebench/search_gc_jit.html), in nightlies?